orphics/maps.py

```diff
@@ -8,6 +8,7 @@
 import math
 from scipy.interpolate import RectBivariateSpline,interp2d,interp1d
 import warnings
+from healpix import hp
```

Comment:
did you mean "import healpy as hp" ?

Reply:
Yup, not sure what happened there. pushed the fix.